README.md

```diff
@@ -1,6 +1,7 @@
 # Camunda Modeler Process I/O Specification Plug-in
 
-[![CI](https://github.com/camunda/camunda-modeler-process-io-specification-plugin/workflows/CI/badge.svg)](https://github.com/camunda/camunda-modeler-process-io-specification-plugin/actions?query=workflow%3ACI) [![Compatible with Camunda Modeler version 5](https://img.shields.io/badge/Camunda%20Modeler-5+-blue.svg)](./README.md#compatibility-notice)
+[![CI](https://github.com/camunda/camunda-modeler-process-io-specification-plugin/workflows/CI/badge.svg)](https://github.com/camunda/camunda-modeler-process-io-specification-plugin/actions?query=workflow%3ACI) [![Compatible with Camunda Modeler version 5](https://img.shields.io/badge/Modeler%20Version-5+-blue.svg)](./README.md#compatibility-notice) [![Plugin Type](https://img.shields.io/badge/Plugin%20Type-bpmn%20(platform)-green.svg)](#)

```

Comment:
Since this is end-user facing and not technical, what about:
```suggestion
[![CI](https://github.com/camunda/camunda-modeler-process-io-specification-plugin/workflows/CI/badge.svg)](https://github.com/camunda/camunda-modeler-process-io-specification-plugin/actions?query=workflow%3ACI) [![Compatible with Camunda Modeler version 5](https://img.shields.io/badge/Modeler%20Version-5+-blue.svg)](./README.md#compatibility-notice) [![Plugin Type](https://img.shields.io/badge/Plugin%20Type-bpmn%20(Camunda 7)-green.svg)](#)
```

Reply:
Yes, even better I think